forester-utils/src/indexer/mod.rs

```diff
@@ -217,6 +239,25 @@ pub trait Indexer<R: RpcConnection>: Sync + Send + Debug + 'static {
     fn add_state_bundle(&mut self, _state_bundle: StateMerkleTreeBundle) {
         unimplemented!()
     }
+
+    async fn update_test_indexer_after_append(
+        &mut self,
+        _rpc: &mut R,
+        _merkle_tree_pubkey: Pubkey,
+        _output_queue_pubkey: Pubkey,
+        _num_inserted_zkps: u64,
+    ) {
+        unimplemented!()
+    }
+
+    async fn update_test_indexer_after_nullification(
```

Comment:
these should be part of test indexer not the indexer trait right?

Reply:
Yes, I'll move it there in the test_indexer's PR